evalml/pipelines/components/estimators/regressors/arima_regressor.py

```diff
@@ -61,6 +61,9 @@ class ARIMARegressor(Estimator):
     supported_problem_types = [ProblemTypes.TIME_SERIES_REGRESSION]
     """[ProblemTypes.TIME_SERIES_REGRESSION]"""
 
+    max_rows = 1000
+    max_cols = 7
```

Comment:
Not a blocker but is there a reason why this is 7 and not like 5 or 10? Just very curious haha

Reply:
Nope, no strong motivation behind it. It was just a bit of a bump down, and without a more comprehensive set of datasets there was no way to test where the best line in the sand would be. Since these results seemed ok, I just stuck with it.